jlm/rvsdg/Makefile.sub

```diff
@@ -36,6 +36,7 @@ librvsdg_HEADERS = \
 	jlm/rvsdg/notifiers.hpp \
 	jlm/rvsdg/bitstring/constant.hpp \
 	jlm/rvsdg/bitstring/slice.hpp \
+	jlm/rvsdg/bitstring/arithmetic-impl.hpp \
```

Comment:
Could we align with the new naming convention here, calling it `ArithmeticImplementation.hpp` or so.

Reply:
Would it not be better to bulk-rename all files later?

Reply:
Sure, let's leave it as it is for now.